translations/en-us.yaml

```diff
@@ -55,3 +55,14 @@ oss-components:
     error:
       title: Error
       subtitle: Failed to copy to your clipboard. Please try again.
+  post_types:
+    facebook_status: Facebook Status
+    tweet: Tweet
+    pin: Pin
+    instagram_media: Instagram Media
+    tiktok_video: Tiktok Video
+    youtube_video: Youtube Video
+    article: Article
+    story: Story
+    twitch_stream: Twitch Stream
```

Comment:
Don't know what the others think (@JulienVannier66 @phndiaye) but maybe we should let the parent assign or not a tooltip.
We might have a case where a SocialMediaBadge is not yet selected and the products wants us to put "Click on this badge to filter by Instagram posts".